### PR TITLE
fix: Weird transition in group info

### PR DESCRIPTION
**Pull request by przemvs**
https://wearezeta.atlassian.net/browse/WPB-11300

## Description

Fixed transitions after refactor less variables.

## Checklist

- [ ] mentions the JIRA issue in the PR name (Ex. [WPB-423])
- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

[WPB-423]: https://wearezeta.atlassian.net/browse/WPB-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/style/panel/panel.less [hunk 1]
```diff
@@ -81,7 +81,7 @@
     &--move-in--right {
       & .panel__content,
       & .panel__footer {
-        transform: translateX(-var(--panel-width));
+        transform: translateX(calc(var(--panel-width) * -1));
```
Comment (aweiss-dev) [inline src/style/panel/panel.less:84]: * -1 is a hack? what does it do? :D

**Review by aweiss-dev — APPROVED** [review]

File: src/style/panel/panel.less (hunk 1 above)
Comment (przemvs) [reply src/style/panel/panel.less:84]: `-var(--panel-width))` - this code doesn't work for negative values so we have to calc * -1.

Comment (e-maad) [reply src/style/panel/panel.less:84]: Why not create another variable like:
`--nagative-panel-width: -304px;`
Just a suggestion.

File: src/style/list/list.less [hunk 2]
```diff
@@ -250,7 +250,7 @@ body.theme-dark {
   &:hover {
     .right-column > .icon-more {
       opacity: 1;
-      transition-delay: calc(var(--animation-timing-faster) + 0.05);
+      transition-delay: calc(var(--animation-timing-faster) + 0.05s);
```
Comment (e-maad) [inline src/style/list/list.less:253]: We have these variables for animation-timing:
```
  --animation-timing-faster: 0.15s;
  --animation-timing-fast: 0.35s;
```

Can we create a new one with `0.20s`?

**Review by e-maad — COMMENTED** [review]
I would prefer to make new variables instead of using calc.

File: src/style/list/list.less (hunk 2 above)
Comment (przemvs) [reply src/style/list/list.less:253]: Ahh... I already merged :( But this is a delay.

**Pull request merged**